tests/pr_test.go

```diff
@@ -170,6 +170,43 @@ func TestRunExistingVPCExample(t *testing.T) {
 		output, err := options.RunTestConsistency()
 		assert.Nil(t, err, "This should not have errored")
 		assert.NotNil(t, output, "Expected some output")
+
+		// test for flow logs with encrypted bucket, new vpc will be created but existing cos instance will be used
```

Comment:
We should be using sub tests for this scenario - see https://github.ibm.com/GoldenEye/issues/issues/12430

However you don't need to create any resources for this test - you can use existing HPCS and COS instance and create a brand new test for this use case